apps/src/tutorialExplorer/filterHeader.jsx

```diff
@@ -119,7 +119,7 @@ const FilterHeader = React.createClass({
                   style={styles.select}
                   className="noFocusButton"
                 >
-                  <option disabled value="default">{i18n.filterHeaderDefault()}</option>
+                  <option disabled hidden value="default">{i18n.filterHeaderDefault()}</option>
```

Comment:
No, it's just used as a kind of caption for the dialog but we don't want it to be one of the selectable options.